src/System.Diagnostics.DiagnosticSource/tests/System.Diagnostics.DiagnosticSource.Tests.builds

```diff
@@ -7,6 +7,10 @@
       <OSGroup>Windows_NT</OSGroup>
       <TestTFMs>netcore50;net46</TestTFMs>
     </Project>
+    <Project Include="System.Diagnostics.DiagnosticSource.Tests.csproj">
```

Comment:
Correct .builds files are no longer used. We just haven't deleted them all yet, we have a clean-up item to delete them all.